data-es6.js

```diff
@@ -18193,6 +18193,7 @@ exports.tests = [
         ie11: true,
         firefox2: true,
         chrome5: true,
+        deno1_24: false,
```

Comment:
It's still marked as supported in 1.23-.

Reply:
I changed it to deno1_3, it's not supported since deno 1.2.1 https://github.com/denoland/deno/commit/fa61956f03491101b6ef64423ea2f1f73af26a73
but I'm not sure whether older version should marked as unsupported or not.